opm/parser/eclipse/EclipseState/SummaryConfig/tests/SummaryConfigTests.cpp

```diff
@@ -185,3 +203,39 @@ BOOST_AUTO_TEST_CASE(completions) {
             keywords.begin(), keywords.end(),
             names.begin(), names.end() );
 }
+
+BOOST_AUTO_TEST_CASE(summary_ALL) {
+
+    const auto input = "ALL\n";
+
+    const auto summary = createSummary( input );
+    const auto key_names = sorted_key_names( summary );
+
+    std::vector<std::string> all;
+
+    for(const std::string& keyword: SummaryConfig::getAllExpandedKeywords()) {
+        if(keyword[0]=='F') {
+            all.push_back(keyword);
+        }
+        else if (keyword[0]=='G') {
+            auto kn = keyword + ": ";
```

Comment:
Hmmmm - this *should* in my opinion fail the test; there should not be a space after the ":"